bn.tex

```diff
@@ -0,0 +1,1913 @@
+\documentclass[synpaper]{book}
```

Comment:
Do we really have to touch this file?

Reply:
no, we don't

Reply:
> no, we don't

Okay, great. Can you please remove it from commit then?

Reply:
How to do it right?

Reply:
Do I need to start over from first step in guide ?

Reply:
> Do I need to start over from first step in guide ?

You mean Hunter documentation? No :) It's a pure Git task.

> How to do it right?

I guess there are a lot of ways but if I will do similar thing I will:
* create another branch for pull request
* switch to new branch
* cherry-pick commits one by one
* stop on the commit that introduce `bn.tex`, do `git rm bn.tex` and `git commit --amend`, effectively this step will remove `bn.tex`
* cherry-pick the rest